### PR TITLE
Plain java script with objects

### PR DESCRIPTION
**Pull request by Mujeeb4582**
## Files Added
- index.html
- script.js
- Linters setup

## Tasks completed

- Create a collection that keeps a list of books (hint: you can use an array of objects for that).
- Create a function to add a new book to the collection, with title and author.
- Create a function to remove a book from the collection
- Display all books saved in the collection in the top part of the page.
- Make sure that when a user clicks the "Add" button:
- A new book is added to the collection.
- The new book is displayed in the page.
- Make sure that when a user clicks the "Remove" button:
- The correct book is removed from the collection.
- The correct book disappears from the page.
- Make sure that data is preserved in the browser's memory by using [localStorage]

## GitHub Tasks
- Update the Readme file and make it professional
- MIT License
- create the PR for review
- checking linters error

## Note:
- If you don't understand anything, Please try to reach us, we love to get feedback.
- Thank you for your time.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  // Add object to the array
+  booksList.push(book);
+
+  // clear the input field on browser
+  addBookForm.bookTitle.value = '';
+  addBookForm.bookAuthor.value = '';
+
+  localStorage.setItem('list', JSON.stringify(booksList));
+
+  // createListItem(bookName,bookAuthorName);
```
Comment (Wes-Isaac) [inline script.js]: - [x] Great job so far, Please remove commented code to make you code-base clean.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @Mujeeb4582 & @Cluab, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Great PR description 👍.
- Project works properly 🚀.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          class="author"
+          name="bookAuthor"
+          type="text"
+          placeholder="Author"
+          required /><br /><br />
+        <button type="submit">Add</button>
+      </form>
+    </section>
+
+    <!-- <script src="./scriptv2.js"></script> -->
```
Comment (Qoosim) [inline index.html:36]: - [OPTIONAL] Kindly remove the commented code as to make your code clean.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada:  :heavy_check_mark: 

Hi @team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_**kindly check some optional comments I left under the review**_ :pray: 

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 
-   Fait README file. :heavy_check_mark: 
-   Descriptive PR title and summary. :heavy_check_mark: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [OPTIONAL] Your README file does not look professional enough. Let's try to form the habit of enduring to make a good README file to every project we build as to train ourselves for the future purpose.

File: index.html (hunk 2 above)
Comment (Mujeeb4582) [reply index.html:36]: okey

**Pull request merged**